run_services.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Start the first process to connect to database 
+python3 connect.py
```

Comment:
It's a bit strange to me that this is needed, since the esi-leap services should also just connect to the database. What behavior do you see if you take this out?

Reply:
No, Actually that's for my testing purpose in an earlier phase. Yes, we don't need this I will remove it.